### ISSUE TITLE
does relaton-iec support an SRD "system-reference-deliverable" doc type?

### ISSUE THREAD

**Issue by ReesePlews** [relaton/relaton-iec#60]
hello i am checking the messages one of my projects and  i have a question if the system-reference-deliverable is supported by relaton-iec?  this issue does not presently cause an error and the document is rendered correctly.

i have copied the following messages:

```
[relaton-iso-bib] WARNING: invalid doctype: `system-reference-deliverable`
[relaton-iso-bib] Allowed doctypes are: `international-standard`, `technical-specification`, `technical-report`, `publicly-available-specification`, `international-workshop-agreement`, `guide`, `amendment`, `technical-corrigendum`, `directive`
[relaton-iec] (IEC SRD 63188:2022) Found: `IEC SRD 63188:2022`
```

the document i am referencing is:
https://webstore.iec.ch/publication/62092

my code to reference is is:
* [[[IEC_SRD_63188_2022,IEC SRD 63188:2022]]]

the pdf render is:

![image](https://github.com/relaton/relaton-iec/assets/35089343/ad95d7b5-a81f-4a63-89ab-8b3ba30d6cbd)

there is no hurry to check this. thank you.

**Comment by ronaldtse** [comment]
Thanks @ReesePlews ! @andrew2net can you help refactor `relaton-iso-bib` a bit so it stops generating errors for non-ISO document types? Seems that this is a repeating issue that IEC document types are raising errors. Thanks!

**Comment by andrew2net** [comment]
Fixed in v1.18.1
